run_examples.py

```diff
@@ -0,0 +1,26 @@
+import os
+from pathlib import Path
+import subprocess
+
+
+def run_python_files(directory):
+    # Recursively walk through the directory
+    for root, _, files in os.walk(directory):
+        for file in files:
+            # Check if it's a Python file
+            if file.endswith(".py"):
+                file_path = os.path.join(root, file)
+                print(f"Running: {file_path}")
+                try:
+                    # Use subprocess to run the Python file
+                    result = subprocess.run(["python", file_path], capture_output=True, text=True)
```

Comment:
Why do you want to capture the output here?

Reply:
Just to keep the output clean